src/odemis/acq/stitching/_weaver.py

```diff
@@ -198,64 +190,67 @@ def getFullImage(self):
                    max(b[2] for b in tbbx_px), max(b[3] for b in tbbx_px))
 
         assert gbbx_px[0] == gbbx_px[1] == 0
-
-        # TODO: warn if the global area is much bigger than the sum of the tile
-        # area
+        if numpy.greater(gbbx_px[-2:], 4 * numpy.sum(tbbx_px[-2:])).any():
+            # Overlap > 50% or missing tiles
+            warn("Global area much bigger than sum of tile areas")
+
+        # Weave tiles by using a smooth gradient. The part of the tile that does not overlap
+        # with any previous tiles is inserted into the part of the
+        # ovv image that is still empty. This part is determined by a mask, which indicates
+        # the parts of the image that already contain image data (True) and the ones that are still
+        # empty (False). For the overlapping parts, the tile is multiplied with weights corresponding
+        # to a gradient that has its maximum at the center of the tile and
+        # smoothly decreases toward the edges. The function for creating the weights is
+        # a distance measure resembling the l5-norm, i.e. a rectangle with smooth edges.
+        # The part of the overview image that overlaps with the new tile is multiplied with the
+        # reverse of the weights (1 -  weights) and the weighted overlapping parts of the new tile and
+        # the ovv image are added, so the resulting image contains a gradient in the overlapping regions
+        # between all the tiles that have been inserted before and the newly inserted tile.
 
         # Paste each tile
         logging.debug("Generating global image of size %dx%d px",
                       gbbx_px[-2], gbbx_px[-1])
         im = numpy.empty((gbbx_px[-1], gbbx_px[-2]), dtype=tiles[0].dtype)
         # Use minimum of the values in the tiles for background
-        im[:] = 0
-        mask = numpy.empty((gbbx_px[-1], gbbx_px[-2]), dtype=tiles[0].dtype)
-        mask[:] = 0
+        im[:] = numpy.amin(tiles)
+        
+        # The mask is multiplied with the tile, thereby creating a tile with a gradient
+        mask = numpy.zeros((gbbx_px[-1], gbbx_px[-2]), dtype=numpy.bool)
 
         for b, t in zip(tbbx_px, tiles):
             # Part of image overlapping with tile
             roi = im[b[1]:b[1] + t.shape[0], b[0]:b[0] + t.shape[1]]
             moi = mask[b[1]:b[1] + t.shape[0], b[0]:b[0] + t.shape[1]]
 
             # Insert image at positions that are still empty
-            roi[moi == 0] = t[moi == 0]
+            roi[-moi] = t[-moi]
```

Comment:
Change the `-` to a `~`. Apparently, they both work on numpy boolean, but it's slightly more intuitive what `~` does.